docusaurus/docs/Android/04-compose/04-message-components/04-message-composer.mdx

```diff
@@ -29,95 +22,51 @@ The easiest way to use the `MessageComposer` is by combining it with the rest of
 ```kotlin
```

Comment:
"combining it with the rest of our components", but in the example below there are no other components

Reply:
Ah, as see item #5, good